### DISCUSSION TITLE
FilePicker's save_file method not working as expected

### DISCUSSION THREAD

**Discussion by mvwilpe** [flet-dev/flet#3930] [General]
It seems that FilePicker's the save_file isn't working as expected.
e.g.
file_picker.save_file(
   dialog_title="this title is ignored",
   initial_directory="/this/works",
   file_name="this name is ignored",
   file_type=ft.FilePickerFileType.IMAGE
 )

So the file name is ignored, (i.e. left blank in the OS file dialog) 
but if the file_name specifies a full path, the specified path for the initial_directory is ignored (but name is still empty).
Also setting the dialog_title  has no effect but perhaps thats a Linux thing?
(file type works, saving the file - after the name is entered  manually in the file dialog - works)

(using Ubunt 24.04)

**Comment by ndonkoHenri** [comment]
Works normally on mac:
![image](https://github.com/user-attachments/assets/a06d448e-285d-4f69-879d-04312cb1aba1)

Might be a Linux-specific issue?

**Comment by mvwilpe** [comment]
Then probably an Ubuntu 24.04 issue since it used to work on 23.11.
